gulpfile.js

```diff
@@ -55,7 +55,7 @@ task('copy-i18n', () => {
 task('styles-global', () => {
     return src(['styles/styles.scss', 'styles/fonts.scss'])
         .pipe(
-            sass().on('error', function (error) {
+            sass({silenceDeprecations: ['legacy-js-api']}).on('error', function (error) {
```

Comment:
Why not upgrade for latest api? We did it in app-builder already for example

Reply:
Upgrade what? This is the issue within the `gulp-dart-scss` package.

Reply:
Updated